src/middlewared/middlewared/alembic/versions/12.0/2020-06-24_17-14_cleanup-ad-parameters.py

```diff
@@ -0,0 +1,29 @@
+"""cleanup AD parameters
+
+Revision ID: 25962b409a1e
+Revises: 8ac8158773c4
```

Comment:
New revisions have been merged since then

Reply:
Thanks for pointing that out.